browser/ui/views/tabs/brave_tab_style_views.inc.cc

```diff
@@ -122,13 +137,27 @@ SkPath BraveVerticalTabStyle::GetPath(
           gfx::OutsetsF::VH(brave_tabs::kHorizontalTabVerticalSpacing * scale,
                             brave_tabs::kHorizontalTabGap / 2 * scale);
 
+      bool frame_condensed =
+          IsBrowserFrameCondensed(tab()->controller()->GetBrowser());
+
       // We should only extend the hit test bounds into the top margin if the
       // window is maximized or in fullscreen mode. Otherwise, we want the space
       // above the visual tab shape to be available for window-dragging.
-      if (!ShouldExtendHitTest()) {
+      if (!frame_condensed) {
```

Comment:
Just curious - this change makes difference?
It seems `ShouldExtendHistTest()` and `IsBrowserFrameCondensed` both are same?

Reply:
They are almost the same, except for the case on Linux with a tiling window manager, where the the window is neither maximized nor fullscreen, but it's frame is collapsed and the child views extend to the edges of the screen.